src/renderer/utils/TestSequenceProjectHandler.ts

```diff
@@ -0,0 +1,335 @@
+// Exposed function in this file should alway follow the following pattern:
+// Take the project as an Input.
+//   1. Always valide the state of the proejct
+//   2. Do the operation
+//   3. Update the test sequencer
+//   4. Return a Result<T, Error>
+// Note: All path in a project should be using '/' for cross-platform compatibility
+
+// Note: Toast.promise doesn't work with neverthrow -----
```

Comment:
toast.promise uses PromiseT type, which allows a regular Promise, are you sure this is true?

Reply:
Yeah, the toast.promise do a bunch of post-processing, even for regular Result type, the Ok=False is interpreted like an Https error :/ Already raise an issue